stack/python/stack.py

```diff
@@ -0,0 +1,52 @@
+class Stack:
+    """
+    Class for operations related to stack.
+    """
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmplnn5ba1l/stack/python/stack.py
+++ b/tmp/tmplnn5ba1l/stack/python/stack.py
@@ -2,6 +2,7 @@
     """
     Class for operations related to stack.
     """
+
     def __init__(self):
         """
         Constructor.
```